bin-src/trim-stats-file.test.ts

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable prettier/prettier */
+import { execSync } from 'child_process';
+import { main } from './trim-stats-file';
+// eslint-disable-next-line jest/no-mocks-import
+import * as trimmedStats from './__mocks__/previewStatsJson/preview-stats.trimmed.json';
+// eslint-disable-next-line jest/no-mocks-import
+import * as baseTrimmedStats from './__mocks__/previewStatsJson/baseline-preview-stats.trimmed.json';
+
+describe('Trim Stats File', () => {
+    it('returns expected file output', async () => {
+        execSync('yarn build-storybook --webpack-stats-json', { encoding: 'utf-8' });
```

Comment:
I think these tests should probably be reworked a little because right now they execute the actual `build-storybook` command as well as `chromatic trim-stats-file`. Probably what you might want is to, or rather, what I'd suggest is to:

- Store your `baseTrimmedStats` as mocks
- Run `yarn build-storybook --webpack-stats-json`, and store that value as mock for Storybook's `preview-stats.json`
- Mock file system so the `./storybook-static/preview-stats.json` path always returns that mock 
- Spy on `outputFile` from `fs-extra`
- run `await main(['./storybook-static/preview-stats.json'])`;
- Use that `outputFile` spy and expect it to be called with `baseTrimmedStats`

Reply:
Thanks, Yann.

After our conversation today, I rewrote the test and ran the file comparison in VSCode to verify the changes are non-breaking.